pom.xml

```diff
@@ -1476,7 +1476,7 @@
                         <environmentVariables>
                             <KYUUBI_WORK_DIR_ROOT>${project.build.directory}/work</KYUUBI_WORK_DIR_ROOT>
                         </environmentVariables>
-                        <systemProperties>
+                        <systemPropertyVariables>
```

Comment:
```
Warning:  Parameter 'systemPropertyVariables' is unknown for plugin 'scalatest-maven-plugin:2.2.0:test (test)'
```
Revert changes on scalatest-maven-plugin's configs.

The warning you are fixing comes from maven-surefire-plugin.

Reply:
Thanks for pointing it out ,I corrected it.😊